requirements/optional.txt

```diff
@@ -1,2 +1,4 @@
 matplotlib>=2.0.0
 dipy>=1.0.0
+opencv-python
```

Comment:
Do we still need opencv?

Reply:
Ok,  It seems that there are no significant improvements. I will remove and leave only the option with the Pillow lib.

```python
import cv2
import io
from PIL import Image 
import numpy as np
import pytest


def convert_pil(image):
    image_encoded = Image.fromarray(image, mode="RGB")
    bytes_img_data = io.BytesIO()
    image_encoded.save(bytes_img_data, format='jpeg')
    bytes_img = bytes_img_data.getvalue()

def convert_cv2(image):
    # preserve width and height, flip B->R
    image = image[:, :, ::-1]
    image_encoded = cv2.imencode('.jpg', image)[1]
    bytes_img = image_encoded.tobytes()

@pytest.mark.parametrize("what", ["cv2", "pil"])
@pytest.mark.benchmark(
    min_rounds=5,
    disable_gc=True,
    warmup=True
)
def test_cv_vs_pil(benchmark, what):
    image = np.random.normal(size=(1024, 2048, 3)).astype(np.uint8)
    if what=='cv2':
        benchmark(convert_cv2, image)
    else:
        benchmark(convert_pil, image)
```

Reply:
Done at 609a439a5e1ae42108ccbf4d49eb9a635f539f29